src/main/java/net/dv8tion/jda/api/entities/TextChannel.java

```diff
@@ -22,15 +22,15 @@
 import javax.annotation.Nonnull;
 
 /**
- * Represents a Discord Text GuildChannel.
- * <br>Adds additional functionality and information for text channels in Discord.
+ * Represents a standard Discord Text GuildChannel.
+ * <br>Adds additional functionality and information for text channels in Discord,
+ * on top of the common functionality present in other guild message channels.
```

Comment:
As a note, the docs here and on NewsChannel are pretty wildly different. We might want to bring them in line with eachother.